lib/resources/r4/related_person.yaml

```diff
@@ -147,8 +147,9 @@ fields:
     }
   note: |
     <ul>
-      <li>The <code>relationship.coding</code> must contain exactly one proprietary system.</li>
-      <li>The <code>relationship.coding</code> cannot contain a standard system.</li>
+      <li> The <code>relationship.coding</code> must contain exactly one proprietary system. </li>
+      <li> The <code>relationship.coding</code> cannot contain a standard system. </li>
+      <li> When specifying a period, the fields must include a time component with a timezone. </li>
```

Comment:
let's keep the original `period` keyword notation throughout this update -
e.g.
```
When specifying a `period`, the fields must include a time component with a timezone.
```

Reply:
This was mentioned on the jira to have it like this Do we need to change it @alexbeckner1

Reply:
+1. Please update to 'period' to make it stand out.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/650/commits/bf850f2bd239f72b66e35382ed7ef3dadbf63f05